nixos/modules/services/logging/beats/apm-server.nix

```diff
@@ -0,0 +1,79 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  beatslib = import ./lib.nix { inherit lib pkgs; };
+
+  mkApmServerConfig = cfg: {
```

Comment:
No need to define functions when you only use them for one argument, just define the result directly.